utility/src/main/java/org/oppia/util/firebase/CrashLoggerImplementation.kt

```diff
@@ -0,0 +1,16 @@
+package org.oppia.util.firebase
+
+import com.google.firebase.crashlytics.FirebaseCrashlytics
+import javax.inject.Singleton
+
+/** Logger for providing custom crash reporting to Firebase Crashlytics. */
+@Singleton
+open class CrashLoggerImplementation(
```

Comment:
This is one implementation of crash logger, the fake is another. In this case, this isn't a fake logger and instead relies on a real backing implementation (Firebase). I suggest naming this accordingly, e.g. ``FirebaseCrashLogger.``